CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 Added
 -----
+- added argument ``--rasa-x-port`` to specify the port of Rasa X when running Rasa X locally via ``rasa x``
```

Comment:
should we also add this to the docs somewhere? (e.g. as a note)

Reply:
We don't show any of the options you have for `rasa x`. What do you think, should we add them here: http://rasa.com/docs/rasa-x/installation-and-setup/? Similar to http://rasa.com/docs/rasa/user-guide/command-line-interface/. Or do you think we should just add the most important ones?

Reply:
Added it to http://rasa.com/docs/rasa/user-guide/command-line-interface/

Reply:
sounds great, thanks :+1: